crates/toolchain/edsl-kernel/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+use p3_baby_bear::BabyBear;
+
+use crate::parse_compiler_output::CompiledKernel;
+
+mod execute;
+mod generate_edsl;
+mod parse_compiler_output;
+mod parse_kernel;
+mod transportation;
+
+const IMMEDIATE_ADDRESS_SPACE: usize = 0;
+const RUST_REGISTER_ADDRESS_SPACE: usize = 1;
+const KERNEL_ADDRESS_SPACE: usize = 5;
+const LONG_FORM_INSTRUCTION_INDICATOR: u32 = (1 << 31) + 115115115;
```

Comment:
these should be defined in a common crate that can be imported by both this crate and the transpiler crate